Buildings/Experimental/OpenBuildingControl/CDL/SetPoints/CalendarTime.mo

```diff
@@ -0,0 +1,360 @@
+within Buildings.Experimental.OpenBuildingControl.CDL.SetPoints;
+model CalendarTime
+  "Computes the unix time stamp and calendar time from the simulation time"
+                   //fixme issues zeroTim==Buildings.Utilities.Time.Types.ZeroTime.Custom, lots of hardcoded stuff
+
+  parameter Buildings.Utilities.Time.Types.ZeroTime zerTim
+    "Enumeration for choosing how reference time (time = 0) should be defined";
+  parameter Integer yearRef(min=firstYear, max=lastYear) = 2016
+    "Year when time = 0, used if zerTim=Custom"
+    annotation(Dialog(enable=zerTim==Buildings.Utilities.Time.Types.ZeroTime.Custom));
```

Comment:
`Buildings.Utilities.Time.Types.ZeroTime` also needs to be copied to `OpenBuildingControl.CDL` so that the code is self-contained.